README.md

```diff
@@ -17,6 +39,10 @@ You can reach the maintainers of this project at:
 - [Slack](https://kubernetes.slack.com/messages/sig-contribex)
 - [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-contribex)
 
+## Logo
+
+* The LWKD logo was designed and donated by [GraphArtgency](https://www.graphartgency.com/). Contact them for community-friendly graphic design!
```

Comment:
Looks like the website of graphartgency.com doesn't exist/work anymore

Reply:
Ah, bummer, removing link.